pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [build-system]
 requires = [
+    "cmake<3.23",
```

Comment:
I think we can remove the version specifier here since we're now using more up to date manylinux image